evap/evaluation/management/commands/anonymize.py

```diff
@@ -205,8 +216,36 @@ def anonymize_questionnaires(self, lorem_ipsum):
                 text_answer.original_answer = self.lorem(text_answer.original_answer, lorem_ipsum)
             text_answer.save()
 
-    # Returns a string with the same number of words as the given text, but
-    # uses lorem ipsum instead of the actual words.
+        self.stdout.write("Shuffling answer counters...")
+        for contribution in Contribution.objects.all():
+            for questionnaire in contribution.questionnaires.all():
+                for question in questionnaire.questions.all():
+                    rating_answer_counters = RatingAnswerCounter.objects.filter(
+                        contribution=contribution,
+                        question=question
+                    )
+                    answer_count = sum(
+                        [rating_answer_counter.count for rating_answer_counter in rating_answer_counters]
+                    )
+                    answers = [rating_answer_counter.answer for rating_answer_counter in rating_answer_counters]
+                    for answer in answers:
+                        counter = RatingAnswerCounter.objects.get(
+                            contribution=contribution,
+                            question=question,
+                            answer=answer
+                        )
+                        if answer == answers[-1]:
+                            new_count = answer_count
+                        else:
+                            new_count = random.randint(0, answer_count)
```

Comment:
this will result in a much larger number of answers than there was before, is that intended?

Reply:
it doesn't, the answer_count is decreased by the number of "assigned" answers every time

Reply:
ah, right, cool. this will make the first answer have a larger value on average than later answers, and there might be some zeroes at the end, but i guess that doesn't matter.